lib/statesman/adapters/active_record.rb

```diff
@@ -53,7 +53,7 @@ def history
 
       def last(force_reload: false)
```

Comment:
Do we want to make sure the other adapters don't have this problem? The memory adapter is safe:

https://github.com/gocardless/statesman/blob/b962ed219e6b05328694def8f9a1d5a504ec5c2e/lib/statesman/adapters/memory.rb#L31-L33

...but the Mongoid adapter has the same bug:

https://github.com/gocardless/statesman/blob/b962ed219e6b05328694def8f9a1d5a504ec5c2e/lib/statesman/adapters/mongoid.rb#L39-L45